ova/provision.sh

```diff
@@ -25,13 +40,21 @@ echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
 # Dependences
 yum install openssl -y
 
-install_wazuh
-
-elastic_stack_${ELK_MAJOR}
+installPrerequisites
+addWazuhrepo
+addElasticRepo
+installWazuh
+installElasticsearch
+installFilebeat
+installKibana
+checkInstallation
+cleanInstall
 
 rm -rf /vagrant
 
 systemctl stop kibana
+systemctl stop filebeat 
 systemctl stop  elasticsearch
+systemctl enable wazuh-manager
+systemctl is-enabled wazuh-manager
 systemctl stop wazuh-manager
```

Comment:
With Wazuh v4.0.0 the manager is stopped by default, so this line won't be necessary.

Reply:
Done in 599737656bcaaa119a2c30c76c1760fa0590643a